stable/storage-class/README.md

```diff
@@ -42,6 +42,17 @@ cloud:
     - us-central1-c
 ```
 
+#### storageClass.*
+
+The purpose of this section is to specify the storage class parameters. These parameters are applied to all storage classes created by this chart.
+
+The following tables list the configurable parameters for the `storageClass` option:
+
+| Parameter | Description | Default |
+| ----- | ----------- | ------ |
+| `allowVolumeExpansion` | Persistent Volumes can be configured to be expandable. This feature when set to true, allows the users to resize the volume by editing the corresponding PVC object. |`true`|
+| `reclaimPolicy` | PersistentVolumes can have various reclaim policies, including "Retain", "Recycle", and "Delete". For dynamically provisioned PersistentVolumes, the default reclaim policy is "Delete". This means that a dynamically provisioned volume is automatically deleted when a user deletes the corresponding PersistentVolumeClaim. |`Retain`|
```

Comment:
Is the default 'Retain' or 'Delete'?

Reply:
The kubernetes default is Delete. Our default is Retain

Reply:
The suggest a minor change to wording:
"For dynamically provisioned PersistentVolumes, the default kubernetes reclaim policy is "Delete".
NuoDB changes this default to "Retain".

Reply:
thanks!